LayoutTests/imported/w3c/web-platform-tests/FileAPI/Blob-methods-from-detached-frame.html

```diff
@@ -48,6 +49,14 @@
     assert_equals(charCodeBufferToString(charCodeBuffer), "bar");
 }, "arrayBuffer()");
 
+promise_test(async () => {
+    const { bytes } = await BlobPrototypeFromDetachedFramePromise;
+    const blob = new Blob(["bar"]);
+
+    const charCodeBytes = await bytes.call(blob);
+    assert_equals(charCodeArrayToString(charCodeBytes), "bar");
+}, "bytes()");
```

Comment:
Nit (and probably a followup)... it might be nicer to have more complete descriptions of exactly what the aim of the test is, though I can see the rest of the tests are also just the method names.